data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,315 @@ mission "Coalition License Patch"
+			`	Finally, he brings you to a room on the third floor, claiming that your passenger is waiting inside. You enter, only to see a Kimek glancing out the window.`
+			choice
+				`	"So, is my passenger crawling around somewhere? In some corner hidden in a web?"`
+				`	"A Kimek? I thought this was an Arach House."`
+			`	The butler ignores what you said and introduces you to the Kimek in question. "Marry into House Mallob at a young age, miss Pichiie did. A former student on the university here, she was."`
+			`	Finally, the reason why you had to come all the way to this part of <origin> becomes clear; Pichiie mentions she would like to hear your stories about where you first got the Yottrite in the first place before boarding your ship to head to <planet>. You do your best to keep it as brief as possible, but she keeps asking more and more questions ranging from how you acquired a Jump Drive to all the places you've been.`
+			`	About an hour after the sun sets, she's finally satisfied, and after saying her goodbyes to other members of the House, you're both brought to your ship.`
+				accept
+	on complete
+		event "yottrite research break" 165
+		payment 200000
+		conversation
+			`As with the Saryd twins, Lugglop greets Pichiie and tells her of a hotel he prepared for her time here.`
+			`	"Ready, my team is now, Captain," he says, after Pichiie leaves with her luggage. "Enrich and research samples as needed, for the next few months we will. Contact you again, I will, should the need for your services arise once again."`
+			`	He hands you <payment>, and walks away.`
+
+
+
+event "yottrite research break"
+
+
+
+mission "Coalition Yottrite 8"
+	name "One Last Sample"
+	description "Acquire one sample of Yottrite and deliver it to <destination> so that Lugglop has some for his presentation to the Coalition's scientific community."
+	landing
+	to offer
+		has "event: yottrite research break"
+	source
+		government "Coalition" "Heliarch"
+	destination "Pelubta Station"
+	on offer
+		conversation
+			`When you land, you receive a message from Lugglop, the Arach who you brought several Yottrite samples to.`
+			`	"Captain, some major findings, we have. Preparing a presentation about our research, to present in Ring of Wisdom, we are," he says. "Only, run out of pure samples, we have. Bring us one once more, would you?"`
+			choice
+				`	"Okay, I'll bring you a piece of Yottrite once again."`
+				`	"Sorry, but I'm not mining that any more."`
+					decline
+			`	He thanks you and says he and the rest of the team will be waiting in <destination>. "Moved here about halfway through the project, we did, to make use of the station's facilities."`
+				accept
+	on complete
+		outfit "Yottrite" -1
+		payment 220000
+		conversation
+			`Lugglop pays you <payment> as you deliver the Yottrite, accompanied by a Heliarch agent.`
+			`	"Finish preparing our presentation, we now must. If to help us further, you wish, in the spaceport meet us you must."`
```

Comment:
When you have short on complete conversation like this, you could just use a dialog instead.